tools/dev-doctor/binarycheck.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+
+	"github.com/blang/semver"
+)
+
+// A binaryCheck checks that a binary called name is installed and optionally at
+// least version minVersion.
+type binaryCheck struct {
```

Comment:
(If you decide to adopt the style nit below, then this would also make sense to just call it "binary".)

Reply:
See comment on nit below.